### DISCUSSION TITLE
ESP8266 wifi setup problems

### DISCUSSION THREAD

**Discussion by Quigongentoo** [thorrak/fermentrack#693] [Q&A] [answered]
Hello all,

I have setup Fermentrack on a pi3 with an Uno and it's working fine, thank you for that.  I have recently acquired (3) ESP8266 boards, I have an extra freezer and thought I would turn it into another fermentation station.  I can flash these boards with Fermentrack  and once flashed I can reach them via their setup page on wireless.  I put my wireless credentials in and press the save button then wait.  The wireless connection drops and I cannot see the ESP's wireless AP name anymore.  I assume its connected to my network.  It however is not.  I cannot then get back to the ESP unless I re-flash it with the serial firmware and dig down to reset Wi-Fi in Fermentrack.  Re-flash the board with the Wi-Fi again and try the process over.  I have used 3 different browsers to try and set the Wi-Fi credentials with no luck.  I have use windows and apple machines.  I've also tried all 3 boards.  Could there be something in the Wi-Fi side of the chip that needs updated?  If so, how would one go about this?  Any suggestion would help.  

Thank you

**Comment by thorrak** [comment]
Which version of the firmware are you trying?

**Comment by Quigongentoo** [comment]
Firmware Flashed: BrewPi-ESP8266 - 0.11 - - WiFi

I did try the old version and the experimental.  The old acted the same way, the experimental I couldn’t get to work.

**Comment by Quigongentoo** [comment, marked as answer]
I figured this out...  It did connect, I was looking for the wrong name.  Its working fine now.  It doesn't help that I have 65 wireless devices on my network.  Sorry for the confusion if any.

Thank you for your time.

**Reply by thorrak** [reply]
Any time - glad it worked out!
